src/librustc_codegen_llvm/intrinsic.rs

```diff
@@ -538,9 +538,33 @@ impl IntrinsicCallMethods<'tcx> for Builder<'a, 'll, 'tcx> {
                         return;
                     }
                 }
-
             },
 
+            "round_unchecked" => {
+                if float_type_width(arg_tys[0]).is_none() {
+                    span_invalid_monomorphization_error(
+                        tcx.sess, span,
+                        &format!("invalid monomorphization of `round_unchecked` intrinsic: \
+                                  expected basic float type, found `{}`", arg_tys[0]));
+                    return;
+                }
+                match int_type_width_signed(ret_ty, self.cx) {
+                    Some((width, _signed @ true)) => {
```

Comment:
Is the `_signed @ true` for documentation? (It's something I'd easily remove in a drive-by commit if there wasn't a comment)

Reply:
Yes it’s intended to show what this boolean means. If you prefer I can replace this pattern with `Some((width, true))` and a `# (width, signed)` comment on the line before. Or maybe a single `match` arm with an `if` expression looks better?

Reply:
`, true` seems good; maybe a comment inside the arm clarifying that:
```rust
// This is the signed case.
```

Reply:
I tried it and like the `if` expression better.